### PR TITLE
Report template

### PR DESCRIPTION
**Pull request by LFISHER7**
This replaces 2 actions that created a notebook jupyter notebook and converted it to html with a single action that renders a templated html file. This will make #10 easier and I think it satisfies #25.

Whilst we're here, this also replaces the hardcoded `measure-description` with a more complete description generated from the other parameters provided.

### CODE REVIEW

File: templates/v2/analysis/report_template.html [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+                        {% for row in top_5_1_data %}
+
+                        {% if loop.first %}
+                        <tr>
+                            {% for col in row %}
+                            <th>
+                                {{ col }}
+                            </th>
+                            {% endfor %}
+                            </th>
```
Comment (lucyb) [inline templates/v2/analysis/report_template.html:86]:
```suggestion
                        </tr>
```

File: templates/v2/analysis/report_template.html [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+                        {% for row in top_5_2_data %}
+
+                        {% if loop.first %}
+                        <tr>
+                            {% for col in row %}
+                            <th>
+                                {{ col }}
+                            </th>
+                            {% endfor %}
+                            </th>
```
Comment (lucyb) [inline templates/v2/analysis/report_template.html:111]:
```suggestion
                        </tr>
```

File: templates/v2/analysis/render_report.py [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+            "description": "Index of Multiple Deprivation breakdown",
+            "figure": figure_paths["imd"],
+        },
+        "region": {
+            "title": "Region",
+            "description": "Region breakdown",
+            "figure": figure_paths["region"],
+        },
+    }
+    # open file from roort directory
```
Comment (lucyb) [inline templates/v2/analysis/render_report.py:118]:
```suggestion
    # open file from root directory
```

File: templates/v2/project.yaml.tmpl [hunk 4]
```diff
@@ -106,25 +106,23 @@ actions:
...
+      --codelist-1-name="{{ codelist_1.label }}"
+      --codelist-2-name="{{ codelist_2.label }}"
+      --codelist-1-link="{{ codelist_1.slug }}"
+      --codelist-2-link="{{ codelist_2.slug }}"
+      --time-value="{{ time_value }}"
+      --time-scale="{{ time_scale }}"
+      --time-event="{{ time_event }}"
+      --start-date="{{ start_date }}"
+      --end-date="{{ end_date }}"
+      --num-practices=1000
```
Comment (lucyb) [inline templates/v2/project.yaml.tmpl:124]: What is num-practices and should it be hard-coded?

**Review by lucyb — APPROVED** [review]
This looks really neat and worked fine when I ran it locally. I noticed a few trivial errors, so have made suggested fixes for them. Otherwise, I'm happy for it to be merged.

I'm sure Tom O will have more opinions about the HTML at a future point, but I also think it's been done in a way that he'll be comfortable changing.

File: templates/v2/project.yaml.tmpl (hunk 4 above)
Comment (LFISHER7) [reply templates/v2/project.yaml.tmpl:124]: This is the number of practices that are used to produce the deciles chart. We don't currently produce it, so it's hard-coded for now. I've added #35 to fix this

**Comment by LFISHER7** [comment]
Thanks!

**Pull request merged**